css/index.css

```diff
@@ -1,3 +1,370 @@
+}
+
+.page-content .page-content-section {
+	width: 100%;
+	padding: 5% 0;
+}
+
+.page-content .page-content-section .learn-more-button {
+	margin-top: 1.5vh;
+	border: 1px solid black;
+	background: white;
+	padding: 2% 5%;
+	border-radius: 5px;
+}
+
+.page-content-img {
+	display: block;
+	width: 100%;
+	height: auto;
+}
+
+.page-content .services-content {
+	border: 1px solid black;
+	background: #f4f3f4;
+	padding: 4% 4%;
+	margin-bottom: 5%;
+}
+
+.services-wrap {
+	flex-wrap: wrap;
+	margin-top: 5%;
+}
+
+/* contact */
+.contact {
+	padding: 5% 0;
+	display: flex;
+	flex-direction: column;
+}
+
+.block-address {
+	display:block;
+}
+
+/* footer */
+footer {
+	width: 100%;
+	text-align: center;
+	padding-bottom: 5%;
+}
+
+address {
+	padding-bottom: 1.5vh;
+}
+
+ul {
+	list-style-type: square;
+	margin: 0;
+	padding: 0;
+	list-style-position: inside;
+	margin-top: 1.5vh;
+}
+
+li {
+	padding-bottom: 1.5vh;
+}
+
+li:last-child {
+	padding-bottom: 0;
+}
+
+/* ------------------------ TABLET ------------------------ */
+
+@media (min-width: 500px) {
+	/* navigation and logo at the top of each page */
+	.top-header-container{
+		position: fixed;
+		background: rgb(255, 255, 255, 0.9);
+		z-index: 100;
+		width: 100%;
+		height: 125px;
+		padding-top: 25px;
+	}
+	
+	.top-header-container .top-header .top-header-nav {
+		flex-direction: row;
+		align-content: center;
+		padding: 20px;
+	}
+	
+	.top-header-container .top-header .top-header-logo {
+		border-bottom: none;
+		padding: 0;
+	}
+	
+	.top-header-container .top-header .top-header-nav a {
+		border-bottom: none;
+		font-size: 16px;
+		padding: 0;
+	}
+
+	/* page container */
+	.page-container {
+		padding-top: 160px;
+	}
+
+	.page-container .page-header {
+		padding: 0 0 5%;
+	}
+
+	/* page content section */
+	.page-content {
+		flex-direction: row;
+		justify-content: space-between;
+	}
+
+	.services-content-img {
+		display: block;
+		width: 100%;
+		height: auto;
+		margin-left: 5%;
+		padding-right: 5%;
+	}
+
+	.page-content .two-column {
+		width: 47.5%;
+		padding: 4% 0;
+	}
+
+	.page-content .three-column {
+		width: 30%;
+		padding: 4% 0;
+	}
+
+	/* specific services.html styling */
+
+	.page-content .services-content {
+		padding: 4%;
+	}
+}
+
+/* ------------------------ DESKTOP, LAPTOP ------------------------ */
+
+@media (min-width: 800px) {
+	#entire-page {
+		max-width: 950px;
+		margin: 0 auto;
+	}
+	/* navigation and logo at the top of each page */
+
+	.top-header-container{
+		height: 110px;
+		max-width: 950px;
+		padding-top: 33px;
+	}
+	
+	.top-header-container .top-header {
+		flex-direction: row;
+		align-items: center;
+	}
+	
+	.top-header-container .top-header .top-header-logo {
+		order: 1;
+		width: auto;
+		padding: 0 4%;
+	}
+
+	.top-header-container .top-header .top-header-nav{
+		justify-content: space-between;
+		padding: 0;
+	}
+	
+	.top-header-container .top-header .top-header-nav a {
+		border-bottom: none;
+		font-size: 16px;
+		padding: 0;
+	}
+
```

Comment:
Quality of life suggestion: I noticed the nav bar has hover effect, it would be nice and consistent for the button to follow the same trend.